packages/medusa-plugin-mailchimp/src/api/routes/index.js

```diff
@@ -3,6 +3,7 @@ import bodyParser from "body-parser"
 import middlewares from "../middleware"
 import { getConfigFile } from "medusa-core-utils"
 import cors from "cors"
+import { parseCorsOrigins } from "@medusajs/medusa/dist/utils"
```

Comment:
todo: Correct import
```suggestion
import { parseCorsOrigins } from "@medusajs/medusa"
```

Will probably require, that we expose our utils in `medusa/src/index.js`, but I believe that should be fine.

Reply:
Alternatively, we could move it to `medusa-core-utils`